youtube_dl/YoutubeDL.py

```diff
@@ -683,7 +687,7 @@ def add_default_extra_info(self, ie_result, ie, url):
             'extractor_key': ie.ie_key(),
         })
 
-    def process_ie_result(self, ie_result, download=True, extra_info={}):
+    def process_ie_result(self, ie_result, download=True, extra_info={}, force_generic_extractor=False):
```

Comment:
If we are always setting it to `False` I think we can remove the parameter from process_ie_result until we actually need it.